code/client/src/core/ui/vehicle_debug.cpp

```diff
@@ -100,6 +102,11 @@ namespace MafiaMP::Core::UI {
                 currentVehicle->SetBeaconLightsOn(beaconsLight);
             }
 
+            bool isEngineOn = currentCar->IsEngineOn();
+            if (ImGui::Checkbox("Engine", &isEngineOn)) {
+                currentVehicle->SetEngineOn(isEngineOn, isEngineOn);
+            }
```

Comment:
Just to be sure, getter is on Car and setter on Vehicle?

Reply:
exactly